browser/notifications/brave_alert_dispatcher_mac.mm

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2019 The Brave Software Team. Distributed under the MPL2
+ * license. This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/browser/notifications/brave_alert_dispatcher_mac.h"
+
+#include <memory>
+#include <set>
+#include <string>
+
+#include "base/callback.h"
+#include "base/mac/scoped_nsobject.h"
+#include "base/strings/sys_string_conversions.h"
+#include "chrome/browser/ui/cocoa/notifications/notification_constants_mac.h"
+#include "chrome/browser/ui/cocoa/notifications/notification_builder_mac.h"
+
+@implementation BraveAlertDispatcher {
+  base::scoped_nsobject<NSMutableArray> alerts_;
+}
+
+- (instancetype)init {
+  if ((self = [super init])) {
+  }
+  return self;
+}
```

Comment:
Don't think this is needed if you aren't doing anything in there

Reply:
Removed, thanks